socorro/cron/jobs/bugzilla.py

```diff
@@ -169,74 +134,31 @@ def inner_transaction(
                         bug_id, signature)
                     useful = True
         except SQLDidNotReturnSingleRow:
-            execute_no_results(
-                connection,
-                """
-                INSERT INTO bugs
-                (id, status, resolution, short_desc)
-                VALUES (%s, %s, %s, %s)""",
-                (bug_id, status, resolution, short_desc)
-            )
-            insert_made = True
             signatures_db = []
 
         for signature in signature_set:
             if signature not in signatures_db:
-                if self._has_signature_report(signature, connection):
-                    execute_no_results(
-                        connection,
-                        """
-                        INSERT INTO bug_associations (signature, bug_id)
-                        VALUES (%s, %s)""",
-                        (signature, bug_id)
-                    )
-                    self.config.logger.info(
-                        'new association: %s - "%s"',
-                        bug_id,
-                        signature
-                    )
-                    useful = True
-                else:
-                    self.config.logger.info(
-                        'rejecting association (no reports with this '
-                        'signature): %s - "%s"',
-                        bug_id,
-                        signature
-                    )
-
-        if useful:
-            if insert_made:
-                self.config.logger.info(
-                    'new bug: %s - %s, %s, "%s"',
-                    bug_id,
-                    status,
-                    resolution,
-                    short_desc
+                execute_no_results(
```

Comment:
This if block looks different. It used to only insert into `bug_associations` if `_has_signature_report` returned True, but now it does it all the time.

It looks like you took that method out, but it uses the `reports` table and not the `bugs` table, so I'm puzzled.